### DISCUSSION TITLE
can't find the mainnet file?

### DISCUSSION THREAD

**Discussion by justinbread** [Chia-Network/chia-blockchain#19034] [Support]
I just install the New version of chia in my new PC, then download the checkpoint package as lead ,
after that, i can't find the " ~/.chia/mainnet/db ". in my PC to store the checkpoint date.
![微信截图_20241212090842](https://github.com/user-attachments/assets/8e82e7c6-340e-48e8-a8c4-905056a7c429)
where should i find the file in my PC, did i  miss any moves before i open it?

**Comment by BrandtH22** [comment]
Hey @justinbread , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Can you provide the translation for that error message?

By default the .chia folder is in your users home directory (the folder with your username on it in `C:/Users`). Did you change that default location by chance?

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
